src/restapi/actions/get.ts

```diff
@@ -4,32 +4,32 @@ import { ActionsResponse } from '../../interfaces/actions';
 import { z } from 'zod';
 
 const getActionsSchema = z.object({
-  path: z.string(),
+  actionId: z.string(),
 });
 
 export type ActionsArgs = z.infer<typeof getActionsSchema> & {
   config: PloneClientConfig;
 };
 
 export const getActions = async ({
-  path,
+  actionId,
```

Comment:
I am confused with this one. You get the available actions per path (every path can have different actions)... you do not query actions IDs. Please refer to the restapi documentation.